lib/aws/eks.ts

```diff
@@ -408,139 +408,167 @@ export class EksStack {
         // }
       },
     });
-
+    
+    const sdk_version = "0.27.2";
     const hypersChart = cluster.addHelmChart("HyperswitchServices", {
-      chart: "hyperswitch-helm",
-      repository: "https://juspay.github.io/hyperswitch-helm",
+      chart: "hyperswitch-stack",
+      repository: "https://juspay.github.io/hyperswitch-helm/v0.1.2",
       namespace: "hyperswitch",
       release: "hypers-v1",
       wait: false,
       values: {
         clusterName: cluster.clusterName,
-        services: {
-          router: {
-            image: `juspaydotin/hyperswitch-router:v1.105.0`
-          },
-          producer: {
-            image: `juspaydotin/hyperswitch-producer:v1.105.0`
-          },
-          consumer: {
-            image: `juspaydotin/hyperswitch-consumer:v1.105.0`
-          },
-          controlCenter: {
-            image: `juspaydotin/hyperswitch-control-center:v1.17.0`
-          }
+        loadBalancer: {
+          targetSecurityGroup: lbSecurityGroup.securityGroupId,
         },
-        application: {
-          server: {
-            serviceAccountAnnotations: {
-              "eks.amazonaws.com/role-arn": hyperswitchServiceAccountRole.roleArn,
+        "hyperswitch-app": {
+          loadBalancer: {
+            targetSecurityGroup: lbSecurityGroup.securityGroupId
+          },
+          services: {
+            router: {
+              version: "v1.107.0",
+              image: `juspaydotin/hyperswitch-router:v1.107.0-standalone`,
```

Comment:
Can we change this to `juspaydotin/hyperswitch-router:v1.107.0`?

Reply:
Even in `juspaydotin/hyperswitch-router:v1.107.0`, we still have to enable kms via env params only. Can't we disable this email feature via envs?